contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,150 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+    getPathsForAllAnswers() {
+        const allAnswerPaths = [];
+
+        // Log ids of all answers in section
+        const allAnswers = this.questions.flatMap(question => question.answers.map(answer => answer.id));
+
+        // Log ids of answers used in minimum paths for recommendations
+        const allRecommendationPaths = this.minimumPathsForRecommendations;
+        const answersUsed = Object.values(allRecommendationPaths).flat().reduce((previous, current) => {
+            if (previous.indexOf(current.answer.id) === -1) {
+                previous.push(current.answer.id);
+            }
+            return previous;
+        }, []);
+
+        // Create paths until all answers are used
+        while (answersUsed.length !== allAnswers.length) {
+            let newPath = [];
+            const lastQuestionWithUnusedAnswers = this.questions.findLast(question => question.answers.some(answer => !answersUsed.includes(answer.id)));
```

Comment:
Real, real minor, but if you swapped this line with 184 (and modified 178 accordingly), then there'd be the slightest of performance increases